packages/x-components/src/x-installer/api/__tests__/default-api.spec.ts

```diff
@@ -28,4 +31,32 @@ describe('testing default X API', () => {
     defaultXAPI.addProductToCart(productId);
     expect(listener).toHaveBeenCalledWith(productId);
   });
+
+  it('changes the `SnippetConfig` when calling the `setSnippetConfig` function', async () => {
```

Comment:
I'm missing another test to ensure that non initial values get added reactively. For example, in this current test add the `store` attribute in another step.